sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -701,19 +701,11 @@ object GpuOverrides {
     expr[Literal](
       "Holds a static value from the query",
       ExprChecks.projectNotLambda(
-        TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL + TypeSig.CALENDAR,
+        TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL + TypeSig.CALENDAR
+          + TypeSig.ARRAY.nested(TypeSig.commonCudfTypes + TypeSig.NULL
+          + TypeSig.DECIMAL + TypeSig.CALENDAR),
```

Comment:
I don't think we can support an array of calendars as a literal either.  That needs to be removed.

Reply:
Removed.